docs/devguide/building/dependencies.rst

```diff
@@ -73,6 +73,14 @@ DOC Group :acetaorel:`ace7tao3`
   This also clones the ``master`` branch of MPC as is.
   :ref:`CMake <cmake-building>` will do the same if :cmake:var:`OPENDDS_ACE_TAO_GIT` is set to ``TRUE``.
 
+.. _ace8tao4:
```

Comment:
Above the ACE/TAO versions there's a line that says "There are two versions of ACE/TAO ...". We should probably changes this to something like "These are the versions of ACE/TAO ...".